app/views/people/show.html.erb

```diff
@@ -95,16 +95,11 @@
 
       <% if @person.has_policy_responsibilities? %>
         <div id="policy">
-          <% @person.current_ministerial_roles.each do |role| %>
-            <% if role.published_policies.any? %>
-              <section class="policy">
-                <h1><%= t('policies.heading') %></h1>
-
-                <%= render partial: 'ministerial_roles/policies',
-                           locals: { role: role } %>
-              </section>
-            <% end %>
-          <% end %>
+          <h1><%= t('policies.heading') %></h1>
```

Comment:
This drops the `<section>` which was wrapping this heading. This page was utilising the fact that the HTML5 spec says that the document hierarchy is reset inside section tags. As it now stands the `<h1>` is no longer inside one of those sections. We have tended to move away from this on newer pages on GOV.UK. In this situation though, to avoid refactoring the whole page, I would recommend updating the `<div id="policy">` above to be a section tag instead.